src/__tests__/HistoryTableRow.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import HistoryTableRow from '../components/HistoryTableRow';
+
+const location = {
+  date: 'May 2',
+  name: 'Richmond',
+  aqi: 99,
+  description: 'Great air quality',
+  color: 'green',
+};
+
+function setup() {
+  const props = {
+    location,
+  };
+
+  const enzymeWrapper = shallow(<HistoryTableRow {...props} />);
+
+  return {
+    props,
+    enzymeWrapper,
+  };
+}
+
+describe('HistoryTableRow component', () => {
+  it('renders self', () => {
+    const { enzymeWrapper, props } = setup();
+
+    expect(enzymeWrapper.find('td')).toHaveLength(5);
```

Comment:
generally avoid multiple "expects" in one "it". look at Jest's snapshot feature if what you are trying to capture is a consistent render behavior.